lib/inspec/cli.rb

```diff
@@ -94,6 +94,8 @@ def json(target)
     desc: "For --what=profile, a list of controls to include. Ignore all other tests."
   option :tags, type: :array,
     desc: "For --what=profile, a list of tags to filter controls and include only those. Ignore all other tests."
+  option :"legacy-export", type: :boolean, default: false,
+         desc: "Run with legacy export."
```

Comment:
You'll also need a conditional to choose between calling info() and info_from_parse() around line 140 (or you can refoctor that more sensbly)

Reply:
json is actually just an alias to export, since InSpec 5. https://github.com/inspec/inspec/blob/7c46d14f3b47dd74aea3a0da32c49facb1dbb06f/lib/inspec/cli.rb#L82

Reply:
this is done